addons/mail/models/mail_followers.py

```diff
@@ -35,6 +35,9 @@ class Followers(models.Model):
     subtype_ids = fields.Many2many(
         'mail.message.subtype', string='Subtype',
         help="Message subtypes followed, meaning subtypes that will be pushed onto the user's Wall.")
+    name = fields.Char('Name', compute='_compute_fields')
+    email = fields.Char('Email', compute='_compute_fields')
+    is_active = fields.Boolean('Is Active', compute='_compute_fields', index=True)
```

Comment:
Is is something different than `active` ( / archive mechanism) ? otherwise, should named `active`